### ISSUE TITLE
Replace hero graphics?

### ISSUE THREAD

**Issue by Quintus** [Secretchronicles/TSC#264]
Hi everyone,

this is a bit of a discussion ticket. I’ve not yet come to a conclusion whether we should keep the original SMC hero graphics, which also never have been vectorised. Given our direction and story change, replacing the hero with a figure not smiling all the time (and removing the M from his head) seems apropriate. We should also discuss changing color away from red (→ Mario reference). Generally, the hero graphics as used when powerups are collected are closely modelled after their Mario pendants, so these should also be changed.

With all this changes, it is probably easier to just create entirely new hero graphics.

Valete,
Quintus

**Comment by Bugsbane** [comment]
I'd enjoy doing new hero graphics, but it seems to me, that's it's a pretty big job, with a fairly low priority. If the "M"'s are really the issue, we can fix that far more quickly, just with a bit of painting on the original PNG's. Beyond that, they're original, non-infringing art assets, already working in the game.

The main real downsides as far as I can see is that we don't have the SVG sources, which makes it hard to create new animations going forward.

**Comment by Quintus** [comment]
I think this ticket should be seen in conjunction with what has been suggested by a user [over in the forums](http://forum.secretchronicles.de/topics/209), i.e. the possibility to have a heroine additionally to the hero.

Vale,
Quintus

**Comment by Bugsbane** [comment]
Sure. If we want a heroine as well (which I love the idea of) then it would be easiest to design both hero and heroine together, rather than trying to recreate the look of Maryo as a girl. Overall I'd say that my thoughts are the same though - fabulous idea, but quite a lot of work when we already have a bunch of stuff unfinished (and I say this as the dad of a 5 year old girl).

So basically, yes, I'd love to create new hero + heroine graphics... after we've gotten the real basics cleared up (eg. all SMB potentially infringing assets and the stuff that is just plain missing or damned ugly like overworld ).

**Comment by datahead8888** [comment]
I put the De-Marioization label on this because we are wiping out "M's" from Alex's hat.

**Comment by Bugsbane** [comment]
[Alex frames download link](http://duruz.com/alex.tar.gz)

Alex frames with the "M"'s removed. I didn't do the ice frames as there's no M visible there, or the FLying Alex frames, as to my knowledge this is a feature that isn't implemented in the game anyway (and by the time it is, we may have different character graphics).

![walk_right_2_holding](https://cloud.githubusercontent.com/assets/401733/8404653/991052b0-1e1c-11e5-891e-11305f51b694.png)

**Comment by Quintus** [comment]
I copied to to alexandria: http://team.secretchronicles.de/~quintus/store/alex.tar.gz

Thank you again, Bugsbane. Any chance we can get SVG versions of him? I know, currently there are none, but for future modifications it’d be good if there were. OTOH, I realise that this ticket is about replacing them altogether, so it’d be probably work in vain...

Vale,
Quintus

**Comment by Bugsbane** [comment]
> Any chance we can get SVG versions of him?

Not really. I don't think there were ever SVG versions of him. From what I heard the originals were bitmap. We could try running the bitmaps though Inkscape's tracing feature. That usually gets a pretty close aproximation, but there are usually small differences, especially on shapes with pointy ends (like the shading on the right side of Alex's face in the image above).

TO make these new frames, I just opened the old bitmaps and painted over the M's in Gimp.

**Comment by Bugsbane** [comment]
On a side note, this is a task I don't really have time for now, but would really enjoy doing when I do have some time.

**Comment by datahead8888** [comment]
> On a side note, this is a task I don't really have time for now, but would really enjoy doing when I do have some time.

I would suggest finalizing the next story draft first so that we have a better idea of the character background of Alex.  I need to review some story documents submitted by another person (danfun64), and then we need a team meeting to make a decision.  We also will want your feedback on the story around that time, @Bugsbane, and you certainly will have a lot of say in the new appearance of Alex.

**Comment by Bugsbane** [comment]
So, is there any way of getting notified of meetings without having to be signed up to get every post to the mailing list?

**Comment by datahead8888** [comment]
Meetings are posted at: http://www.secretchronicles.de/en/ when scheduled.

Obviously you would have to have seen a notice to know of the meeting.  We might be able to try to email you individually if you privately give us your email.  Otherwise, we could see if @Quintus has any other thoughts on good ways to communicate with you.

**Comment by datahead8888** [comment]
Meetings are posted at: http://www.secretchronicles.de/en/ when scheduled.

Obviously you would have to have seen a notice to know of the meeting. We might be able to try to email you individually if you privately give us your email. Otherwise, we could see if @Quintus has any other thoughts on good ways to communicate with you. Might a meetings email list or critical announcements list be another good option?

**Comment by Bugsbane** [comment]
Some kind of events mailing list would be fine.I just want to keep my inbox clear, so I avoid mailing lists usually as most of the messages aren't usually personally relevant. Team meetings and version launches is have no problem with though.

**Comment by Quintus** [comment]
Bugsbane <notifications@github.com> writes:
> Some kind of events mailing list would be fine.I just want to keep my
> inbox clear,

I recommend filtering your mail directly on the mail server. I am
subscribed to about 10 mailinglists, and without email filtering I would
get overwhelmed, but this way they’re all nicely sorted into the correct
folders before I even see them. I can then decide which ML to view and
which not to view; for example I ignore most of the traffic on the MinGW
mailinglist, but closely follow Ruby-Talk. My INBOX then only receives
those mails that are not affected by any filters, which are relevant
messages mostly.

If you don’t want to subscribe to the ML (which I still highly
recommend, incombination with mail filtering), you can syndicate the
website’s RSS feed at <http://www.secretchronicles.de/en/feed.atom>. It
contains all news posted to the main site, and meetings are included in
that.

If you want, you can have an `@secretchronicles.de` email
address. alexandria provides mail filtering directly on the server (via
Sieve). Ask @sydneyjd if you’re interested in that.

Valete,
Quintus

**Comment by sydneyjd** [comment]
I for one can say that filters are an awesome email feature also. It 
sorts them into specific folders based on what i have set up. Sometimes 
i get 20 emails, and they all get stuck where i want them, to, as 
Quintus said, look at ( or not look at) based on their needs, and so 
they don't clog my inbox.

-Sydney

**Comment by Bugsbane** [comment]
I have a much simpler and more effective (for me) solution. I don't subscribe to mailing lists. No folders. No setup. No ballooning volumes of storage space. Faster and easier than any of the proposed solutions.

> you can syndicate the website’s RSS feed at <http://www.secretchronicles.de/en/feed.atom>

I <3 my RSS, so this is perfect, as long as the news goes up at least a couple of days beforehand as I don't read my feeds every day.

**Comment by datahead8888** [comment]
Does anyone have a little time to get M'less Alex added to the devel branch?  I'm tied up with another task at the moment, but I wanted to make sure the team was aware this one wasn't committed yet.

**Comment by datahead8888** [comment]
Thanks for committing them, Quintus.

**Comment by datahead8888** [comment]
This overworld graphic still has M's:
https://github.com/Secretchronicles/TSC/blob/a53fac89201051d9effe2e782357217364dc5d31/tsc/data/pixmaps/world/alex/small/down.png

We might try to get the M's removed before the next release.  We can then decide on redoing the graphics altogether in another release.  I'm almost done with a document outlining story principles, which will facilitate discussion of Alex's age and profession, both of which could be included in the new graphics.

**Comment by Quintus** [comment]
Agreed. Maybe something for DarkAceZ to do :-)

Vale,
Quintus

-- 
#!/sbin/quintus
Blog: http://www.guelkerdev.de

GnuPG key: F1D8799FBCC8BC4F

**Comment by DarkAceZ** [comment]
I'll get on it right after work, this weekend.
Do I only need to alter the original SVG's, or will PNG's need to be added to the commit as well?

**Comment by datahead8888** [comment]
If you have the SVG's, alter them as you can.  Otherwise we at least need to edit the PNG's.

**Comment by Quintus** [comment]
DarkAceZ <notifications@github.com> writes:

> I'll get on it right after work, this weekend.
> Do I only need to alter the original SVG's, or will PNG's need to be
> added to the commit as well?

There are no SVGs for this legacy graphic to my knowledge. Altering the
PNGs is enough.

Vale,
Quintus

-- 
#!/sbin/quintus
Blog: http://www.guelkerdev.de

GnuPG key: F1D8799FBCC8BC4F

**Comment by Bugsbane** [comment]
Ugh... Overworld Maryo was actually done by me so I *might* still have the SVG's for this... finding them is another matter though.

**Comment by datahead8888** [comment]
This is the link to the discussion / image:
http://forum.secretchronicles.de/topics/70

**Comment by datahead8888** [comment]
Mispost...deleting...
